### ISSUE TITLE
Wrong memory usage

### ISSUE THREAD

**Issue by juriscorza** [tobi-wan-kenobi/bumblebee-status#416]
#### Summary
Affected module: memory.py (i suppose)

Description:
The module that calculates the ram usage does not work properly, infact it shows much more memory compared to the one actually used.

#### What is the current behaviour and what is the expected behaviour?
free -h | awk '/^Mem:/ {print $3 "/" $2}' shows 776Mi/15Gi
The memory indicated in the bar shows 1.18GiB/15.36GiB

The percentage becomes more and more accurate as the memory used grows.

#### Expected behavior
The correct amount of memory usage.

**Comment by tobi-wan-kenobi** [comment]
Hi,

please have a look at #402, calculating the amount of available memory on a Linux system is non-trivial. Most tools (htop, gnome-system-monitor, etc.) report slightly different values, depending on their definition of "free".

In your particular case, using the column "free" in the output of "free -h" is misleading, if you have it, the column "available" would be better, from the man page:

 Estimation  of how much memory is available for starting new applications, without swap‐
              ping. Unlike the data provided by the cache or free fields, this field  takes  into  ac‐
              count page cache and also that not all reclaimable memory slabs will be reclaimed due to
              items being in use (MemAvailable in /proc/meminfo, available on kernels  3.14,  emulated
              on kernels 2.6.27+, otherwise the same as free)

In short, the column "free" does not for memory that is marked as used, but is reclaimable (and therefore, available to other applications).
